pkg/daemon/gpsd.go

```diff
@@ -176,18 +188,18 @@ func (g *GPSD) CmdRun(stdoutToSocket bool) {
 		g.ProcessStatus(nil, PtpProcessUp)
 		glog.Infof("Starting %s...", g.Name())
 		glog.Infof("%s cmd: %+v", g.Name(), g.cmd)
-		g.cmd.Stderr = os.Stderr
-		var err error
+		g.cmd.Stderr = &filteringStderrWriter{}
+		var cmdErr error
```

Comment:
Not sure of the need for this change but its ok.

Reply:
Oh I just an cmdErr vs err, I was being picky about the name change :P

Reply:
fixed it